src/plugins/intel_cpu/src/nodes/executors/convolution_implementations.cpp

```diff
@@ -64,6 +68,12 @@ static const TypeMapping dnnlConvTypeMapping {
     {{_any, _any, _any, _any},                                {just<f32>(), just<f32>(), just<f32>(), just<f32>()}},
     // @todo explicitly cover configuration limitations for oneDNN on ARM
 };
+
+static const TypeMapping aclLowpConvTypeMapping {
+    // {src, wei, bia, dst}                            pt<src, wei, bias, dst>
+    {{_u8, _u8 | _i8, _i32 | _dynamic, _u8},           {bypass(), bypass(), bypass(), bypass()}},
```

Comment:
So isn't it enough to specify i32 bias here?
Do we really need a check in graph_optimizer?

Reply:
Answered above